scripts/arr.js

```diff
@@ -0,0 +1,18 @@
+
+let arr = [2, 5, 3, 9];
+let result = (arr[0]*arr[1]) + (arr[2]*arr[3]);
+alert(result);
```

Comment:
It should be function since you expect it to be run. And again: use `let` only when needed. Use full naming for variable naming. Array should be given as an argument.

One function do the work (your task), another executes it and interacts with user.
```suggestion

function sumMultiply(array) {
  if (!Array.isArray(array) || array.length !== 4) {
    return 'Array is expected to have 4 numbers';
  }
  
  return array[0]*array[1] + array[2]*array[3];
}

function runSumMultiply() {
  const userArray = prompt('Enter array:', '[2, 5, 3, 9]') || [];
  
  const result = sumMultiply(userArray);
  
  alert(result);
}

runSumMultiply(); // this is the function that should be run on button click

```

Reply:
I don't understand this function sumMultiply(array).
And what does it mean:
Array.isArray(array) ?
array.length ?
Why we make  return 'Array is expected to have 4 numbers';  ?

Reply:
https://frontendrockers.slack.com/archives/D016GU73WRM/p1610839367004200